docs/README.md

```diff
@@ -44,3 +45,26 @@ We recommend the following common development tools:
 * [Mailhog](https://github.com/Chassis/MailHog) - Captures outbound email from Altis and provides a fake inbox
 
 Consult the [Chassis documentation](http://docs.chassis.io/en/latest/extend/) for information about installing additional extensions.
+
+
+## Using HTTPS locally
+
+Local Chassis will generate an SSL certificate you can use to run your local environment over HTTPS. The file will be located in the `/chassis` directory, by default it will be called `altis.local.crt` but if you have customised the `hosts` in `config.local.yaml` it will use the first host name in that list for the file name.
```

Comment:
Is the extension for the cert file `.crt` or `.cert`? Here it's `.crt` but in `$cert_file` in the `Command::secure()` method it's `.cert`.